server/bin/pbench-backup-tarballs

```diff
@@ -271,6 +275,7 @@ def backup_data(lb_obj, s3_obj, config, logger):
             tar = Path(tb).resolve(strict=True)
         except FileNotFoundError:
             tar = None
+            sync.error(dataset, f"tarball link {tb} isn't a real file")
             logger.error("Tarball link, '{}', does not resolve to a real location", tb)
```

Comment:
If it's not a real file, then...it's a virtual file?  a(in il)logical file?  Er....  😆

Is there some benefit to having a different message for the `Sync.error()` from the one used for the logger in the next line?

Reply:
Any interest in improving the `Sync` message, or should I just mark this as resolved?